content/en/docs/score specification/score-spec-reference.md

```diff
@@ -30,27 +30,27 @@ apiVersion: string
 
 metadata:
   name: string
+  annotations: # optional
+    annotations-name: string # optional
 ```
 
-`apiVersion`: the declared Score Specification version.
+`apiVersion`: the declared Score Specification version. Find the current version [here](https://github.com/score-spec/schema/blob/main/score-v1b1.json).
 
-- **Valid options**: `score.dev/v1b1`
-
-`metadata`: the metadata description of your Workload.
-
-- `name`: specifies a string that can describe your Workload.
-
-You can reference any information available in the metadata section too.
+`metadata`: the metadata description of your workload.
+  - `name`: a string that describes your workload.
+  - `annotations`: a set of optional annotations that apply to the workload and can be passed through to the destination runtime.
 
 ### Workload example
 
-The following is a top level description for a Workload.
+The following is a top level description for a workload.
 
 ```yaml
 apiVersion: score.dev/v1b1
 
 metadata:
   name: hello-world
+  annotations:
+    example.com/my-annotation: value
 service:
```

Comment:
Sorry for my piecemeal feedback, just now noticing this: can we reverse the order of `service` and `container` in this section as well for consistency?

Reply:
Good point, changed! ✅